packages/appstore/src/components/add-options/add-options.scss

```diff
@@ -15,11 +17,18 @@
         & > p {
             color: $color-red;
             margin-left: 0.8rem;
+            font-weight: 700;
+            font-size: 1.2rem;
+            line-height: 1.8rem;
         }
     }
     &_description {
         & > p {
             color: $color-grey-1;
```

Comment:
Nitpick: You can also use 'var(--text-less-prominent)' as it will help with dark mode